kubernetes/cray-sysmgmt-health/templates/grok-exporter/deployment.yaml

```diff
@@ -33,6 +33,11 @@ spec:
   selector:
     matchLabels:
       app: grok-exporter
+  strategy:
+    rollingUpdate:
+      maxSurge: 35%
```

Comment:
Just for information in the future, maxSurge and maxUnavailable can also be expressed as an absolute number of pods instead of a percentage (e.g. 1).

Reply:
Sure. Thank you, we will use as absolute number if need be.